openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -76,7 +58,26 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
                         % endif
                     </a>
                 % endif
+                % if settings.FEATURES.get('ENABLE_COURSEWARE_SEARCH'):
+                    <div class="page-header-search">
+                        <form class="search-form input-group" role="search" action="${reverse('openedx.course_search.course_search_results', args=[course_key])}">
+                            <label class="field-label sr-only" for="search" id="search-hint">${_('Search the course')}</label>
+                            <input
+                                    class="field-input input-text search-input form-control"
+                                    type="search"
+                                    name="query"
+                                    id="search"
+                                    placeholder="${_('Search the course')}"
+                            />
+                            <span class="input-group-btn">
+                                <button class="btn btn-outline-primary search-button" type="submit">${_('Search')}</button>
+                            </span>
+                        </form>
+                    </div>
+                % endif
             </div>
+            <h2>${course.display_name_with_default}</h2>
+            <h3>Organization</h3>
```

Comment:
In case we need the translations later.
```suggestion
            <h3>${_('Organization')}</h3>
```

Reply:
It is just a static string put for a developer, who will update it.